ScootBot/Program.cs

```diff
@@ -219,6 +219,9 @@ private static void ProcessCommand(ChatMessage msg, string command)
                 case "poll":
                     result = Poll("list", msg.Sender.Handle);
                     break;
+                case "kill":
+                    while(1);
```

Comment:
I might disagree. This might possibly be a good kill-switch for the day that SCbot becomes an independent agent. At that time, however, we'll have to evaluate the ethics of killing a person (or what might approximate a person).